spec/requests/admin/memberships_spec.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Admin::MembershipsController, type: [:request, :admin] do
```

Comment:
RSpec/FilePath: Spec path should end with `admin/memberships_controller*_spec.rb`. (https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/FilePath)